pkg/utils/litmus/litmusClient.go

```diff
@@ -0,0 +1,17 @@
+package litmus
+
+import (
+	"fmt"
+
+	clientV1alpha1 "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned"
+	"k8s.io/client-go/rest"
```

Comment:
```
	"fmt"

	"k8s.io/client-go/rest"

	clientV1alpha1 "github.com/litmuschaos/chaos-operator/pkg/client/clientset/versioned"
```

Reply:
It was like that before, will update all of them